RSyntaxTextArea/src/main/java/org/fife/ui/rsyntaxtextarea/modes/PythonTokenMaker.java

```diff
@@ -682,6 +682,15 @@ public String[] getLineCommentStartAndEnd(int languageIndex) {
 		return new String[] { "#", null };
 	}
 
+	@Override
+	public boolean getShouldIndentNextLineAfter(Token t) {
+		if (t!=null && t.length()==1) {
+			char ch = t.charAt(0);
+			return ch==':' || ch=='\\';
+		}
+		return false;
+	}
+
```

Comment:
You'll need to add this method to `PythonTokenMaker.flex` as well, as this file is generated. It's technicaally cheating, but if you don't want to work out running JFlex 1.4.1 (the older version this project relies on), you can just add this code snippet to that file in the same relative spot and it's functionally equivalent.

Reply:
OK.  Will commit with other requested changes.